### PR TITLE
[319] Verify drives on frontend

### LINKED ISSUE #319: Browser-side sign validation based on passenger public key.
When passenger signs received hash value and passes that back to driver he will need to verify the signature using passenger's public key stored in local storage.

### PR DESCRIPTION
**Pull request by arturtamborski**
Resolves #319. This code adds color bar as an indication for drives which were verified.

### CODE REVIEW

File: frontend/src/views/DriveFormView.vue [hunk 1]
```diff
@@ -329,6 +337,7 @@ export default {
     ...mapState(namespaces.passengers, {
       passengers: state =>
         (state.data || []).map(p => ({
+          ...p,
```
Comment (pniedzwiedzinski) [inline frontend/src/views/DriveFormView.vue]:
```suggestion
          rsaPubE,
          rsaModulusN
```

File: backend/fleet_management/serializers.py [hunk 2]
```diff
@@ -16,10 +16,13 @@ class Meta:
...
 class UserSerializer(serializers.ModelSerializer):
     groups = GroupSerializer(many=True)
+    rsa_modulus_n = fields.CharField(read_only=True)
+    rsa_pub_e = fields.CharField(read_only=True)
+    rsa_priv_d = fields.CharField(read_only=True)
 
     class Meta:
         model = User
-        fields = ("id", "username", "groups")
+        fields = ("id", "username", "groups", "rsa_modulus_n", "rsa_pub_e", "rsa_priv_d")
```
Comment (pniedzwiedzinski) [inline backend/fleet_management/serializers.py]: Do we need to pass the private key?

Comment (arturtamborski) [reply backend/fleet_management/serializers.py]: Yes, this one is a different serializer, it's used only for getting the details of currently logged in user (via `/api/users/me`). And we want these details but only for the _logged in_ user.
Passenger serializer defined below (used by Drivers) does not expose the private key.

File: frontend/src/views/DriveFormView.vue (hunk 1 above)
Comment (arturtamborski) [reply frontend/src/views/DriveFormView.vue]: I don't quite get how to do it like you suggested. I can't just list `rsaPubE`, `rsaModulusN` like that can I?
I did the usual `rsaPubE: p.rsaPubE`, hope that's what you meant.

Comment (pniedzwiedzinski) [reply frontend/src/views/DriveFormView.vue]: Right, my bad, maybe something like this:

```js
(state.data || []).map(
  ({id, firstName, lastName, rsaModulusN, rsaPubE}) => ({
          value: id,
          text: [firstName, lastName].join(' '),
          rsaModulusN,
          rsaPubE,
  })),
```

Comment (arturtamborski) [reply frontend/src/views/DriveFormView.vue]: whoa, that's wild! :D :dancer: Will test it later and see how it goes

Comment (arturtamborski) [reply frontend/src/views/DriveFormView.vue]: After second thought - don't you think that the current version is good enough?

Comment (pniedzwiedzinski) [reply frontend/src/views/DriveFormView.vue]: it's ok

**Pull request merged**